packages/wrangler/src/bundle.ts

```diff
@@ -60,6 +61,30 @@ export async function bundleWorker(
     format: entry.format,
     rules,
   });
+  const checkForNodeBuiltinsPlugin = {
```

Comment:
hoist this to outside bundleWorker()
```suggestion
 // Add some whitespace and description of what this plugin is 
 /**
 * This plugin's description
 */
  const checkForNodeBuiltinsPlugin = {
```